packages/odata-v2/src/request-builder/update-request-builder.spec.ts

```diff
@@ -208,7 +258,11 @@ describe('UpdateRequestBuilder', () => {
 
   it('update request should contain version identifier when set on entity', async () => {
     const entity = createTestEntity().setVersionIdentifier('not-a-star');
-    const requestBody = { Int32Property: entity.int32Property };
+    const requestBody = {
+      Int32Property: entity.int32Property,
+      StringProperty: null,
```

Comment:
[q] Why are the two options set to null here?

Reply:
Would it be bad?

Yes this changes that all null values are being sent where before none has been sent.

My understanding was that this is nock specific because it limits what the 'server' reacts to. My assumption is that real servers don't care about additional null fields. If the value was null before it does not change anything, if the value was not null before you wanted to set it to null.

Not sure how we could avoid that if we don't add another parameter for 'null fields I want to update'.